roles/backup/tasks/main.yml

```
@@ -0,0 +1,16 @@
+---
+- name: Place backup script
+  template:
+    dest: /root/backup.sh
+    src: backup.sh.j2
+    owner: root
+    group: root
+    mode: 0644
```

```suggestion
    mode: 0755
```
The script must be executable. I'm sorry, I guess, I suggested these wrong permissions.

---

Thanks :)